### ISSUE TITLE
games-engines/ags version bump and missing dependency >=media-libs/aldumb-0.9.3

### ISSUE THREAD

**Issue by tsr** [leycec/raiagent#93]
Trying to install ags:
```
emerge -a games-engines/ags
```

will result in:
```
emerge: there are no ebuilds to satisfy ">=media-libs/aldumb-0.9.3".
```

I can't find media-libs/aldumb in any overlay, so it would be very nice if this library could be include in this overlay.

**Comment by leycec** [comment]
**Fascinating!** Over the slow pressure-cooking of time, I'd forgotten all about Adventure Game Studio (AGS). I'm both delighted and surprised that it's still in use, so I'll resurrect this is soon as I reasonably can. Thanks to the Holidays, that'll probably be more than a few days from now.

**Comment by leycec** [comment]
**Ho, ho, ho.** Merry Christmas, @tsr. I did a *lot* here, so let me know if any of it works as expected. From our `git log` for this commit:

> This commit bumps our Adventure Game Studio (AGS) ebuild to its most stable release as well as EAPI 7 and resolves #93 by rewriting that ebuild's dependency on the recently last-rited `media-libs/aldumb` to the existing `media-libs/dumb[allegro]"` which now provides the equivalent of `media-libs/aldumb`. Thanks to stalwart hero @tsr for reporting this critical breakage!

> This commit also resolves a variety of upstream issues, including forced linking against static rather than shared libraries for the `media-libs/dumb` dependency. Phew.

Good things come to those who submit good issues. That means you! :santa:

**Comment by tsr** [comment]
Works fine, thanks alot!
And Merry Christmas! :santa:
